StencilFrontend.md

```diff
@@ -0,0 +1,353 @@
+
+### Create the Stencil File
+To use templates with a Kitura router, you must create a `.stencil` template file which describes how to embed code into the returned HTML. By default, the Kitura router gets the template files from the `./Views/` directory in the directory where Kitura runs.
+
+1. In the terminal, change into the FoodServer directory:
+```
+cd ~/FoodTrackerBackend/FoodServer
+```
+2. Create the Views folder and change into it:
+```
+mkdir Views
+cd Views
+```
+3. Create a `FoodTemplate.stencil` file.
+We will use terminal commands for file creation but you can use a text editor of your choice.
+```
+cat > FoodTemplate.stencil
+```
+**Note:** You can exit `cat` using `CTRL+D`
+
+4. Add the following Stencil template code to display the meals:
+
+```
+<html>
+There are {{ meals.count }} meals. <br />
+
+{% for meal in meals %}
+    - {{ meal.name }} with rating {{ meal.rating }}. <br />
+    <br />
+{% endfor %}
+</html>
+```
+5. Save the file and exit your text editor. For `cat` use the shortcut:  `CTRL+D`
+
+When the above code is rendered, it will display the total number of meals in your meal store, it will then loop through the meal store displaying each meal name and rating. For more details about Stencil templates see the [Stencil User Guide](http://stencil.fuller.li/en/latest/).
+
+
+### Add the Kitura-StencilTemplateEngine dependency
+[Kitura-TemplateEngine](https://github.com/IBM-Swift/Kitura-TemplateEngine) is a template engine abstraction layer which works alongside a specific template engine, such as [Kitura-StencilTemplateEngine](https://github.com/IBM-Swift/Kitura-StencilTemplateEngine), to allow a user to easily use a template engine in Swift. We need to add the `Kitura-StencilTemplateEngine` dependency to our `Package.swift` file.
+
+1. In the terminal, go to your server's `Package.swift` file.
+```
+cd ~/FoodTrackerBackend/FoodServer
+open Package.swift
+```
+2. Add the `Kitura-StencilTemplateEngine` package:
+```swift
+.package(url: "https://github.com/IBM-Swift/Kitura-StencilTemplateEngine.git", from: "1.8.0")
+```
+3. Change the target for Application to include "KituraStencil":
+```swift
+.target(name: "Application", dependencies: [ "Kitura", "Configuration", "CloudEnvironment", "SwiftMetrics", "Health", "KituraStencil"]),
+```
+
+### Generate your FoodServer Xcode project
```

Comment:
Can't this whole section move to be point 4 in the previous section, such as the following, as the user will have already generated this Xcode project in the previous tutorial or set up?

4. Regenerate your server Xcode project:
```
swift package generate-xcodeproj
```